packages/core/src/components/content-header/content-header.scss

```diff
@@ -17,6 +17,18 @@
     display: flex;
     flex-direction: column;
     flex: 1 1 0;
+    margin-right: 0.5rem;
+
+    .headerTitleRow {
+      display: flex;
+      min-width: 6rem;
+
+      .headerSlot {
+        display: inline-flex;
+        margin-top: 0.375rem;
```

Comment:
Even if most of the time ix-pill will be inserted into the slot, the top margin might get in the way of other elements. IMO we should not set that and leave it to the slot user.

Thoughts @danielleroux?